demo/app/integration_test/openid4ci_test.dart

```diff
@@ -120,10 +120,77 @@ void main() async {
     print("matchedCreds : $matchedCreds");
 
     expect(matchedCreds, hasLength(equals(3)));
+
     expect(matchedCreds[0], equals(credentials[0]));
     expect(matchedCreds[1], equals(credentials[1]));
     expect(matchedCreds[2], equals(credentials[2]));
 
     await walletSDKPlugin.presentCredential();
   });
+
+  testWidgets('Testing openid4vc with a auth code flow', (tester) async {
+
+    const didMethodTypes = String.fromEnvironment("WALLET_DID_METHODS");
+    var didMethodTypesList = didMethodTypes.split(' ');
+    String didMethodType = didMethodTypesList[0];
+    print("wallet DID type : $didMethodType");
+    print("wallet DID Key type : $didKeyType");
+    var didDocData = await walletSDKPlugin.createDID(didMethodTypesList[0], didKeyType);
+    print("wallet didDocData : $didDocData");
+    var didDocEncoded = json.encode(didDocData!);
+    Map<String, dynamic> responseJson = json.decode(didDocEncoded);
+    var didContent= responseJson["did"];
+    print("wallet DID : $didContent");
+
+    const issuanceURL = String.fromEnvironment("INITIATE_ISSUANCE_URLS_AUTH_CODE_FLOW");
+    print("issuanceURLs in test: $issuanceURL");
+
+    const verificationURL = String.fromEnvironment("INITIATE_VERIFICATION_URLS");
+
+      var authCodeArgs =   {
+        "scopes": ["openid","profile"],
+        "clientID": "oidc4vc_client",
+        "redirectURI": "http://127.0.0.1/callback"
+      };
+
+      var initializeResp = await walletSDKPlugin.initialize(issuanceURL, authCodeArgs);
+      var initializeRespEncoded = json.encode(initializeResp!);
+      Map<String, dynamic> initializeRespJson = json.decode(initializeRespEncoded);
+      var authorizationURLLink= initializeRespJson["authorizationURLLink"];
+    debugPrint("-------------------------");
```

Comment:
Did you mean to leave in all these print statements?

Reply:
yea couple of them for CI